opensrp-app/src/main/java/org/smartregister/repository/PlanDefinitionSearchRepository.java

```diff
@@ -95,19 +95,35 @@ public void addOrUpdate(PlanDefinition planDefinition, String jurisdiction) {
     }
 
     public List<PlanDefinitionSearch> findPlanDefinitionSearchByPlanId(@NonNull String planId) {
-        List<PlanDefinitionSearch> planDefinitionSearchSet = new ArrayList<>();
-        String query = String.format("SELECT %s FROM %s WHERE %s=? ", 
-        StringUtils.join(COLUMNS, ","), PLAN_DEFINITION_SEARCH_TABLE, PLAN_ID);
+        List<PlanDefinitionSearch> planDefinitionSearchList = new ArrayList<>();
+        String query = String.format("SELECT %s FROM %s WHERE %s=? ",
+                StringUtils.join(COLUMNS, ","), PLAN_DEFINITION_SEARCH_TABLE, PLAN_ID);
```

Comment:
shouldn't you select `*` instead of passing in columns as you are querying all columns

Reply:
okay updating